app/models/concerns/user_totp_methods.rb

```diff
@@ -20,8 +33,12 @@ def verify_and_enable_totp!(seed, level, otp, expiry)
   end
 
   def enable_totp!(seed, level)
```

Comment:
Discussed with Eric on Friday and decided to keep the methods signature as is for now.

Should we remove `level` in the signature? In production, level is always `ui_and_api`. Only in tests, level is set as something else. As we are now adding the clause where we don't change the level if webauthn is enabled, it can lead to weird behaviour that we let the function accept which level to change it to.